source/docs/quick_start.md

```diff
@@ -10,6 +10,8 @@ $ cd embark_demo
 
 ### Start a blockchain node or let Embark start it for you
 
+Note: starting in Embark 3.1 the command `embark run` will automatically start a blockchain node if one is not already started, you may therefore skip this step.
```

Comment:
This might read a tad bit better:

```
Note: starting in Embark 3.1, the command `embark run` will automatically start a blockchain node if one isn't already running. You may, therefore, skip this step.